### DISCUSSION TITLE
Huge Summer Sale at PrimeStore: Save Up to 50%

### DISCUSSION THREAD

**Discussion by onursenture** [primefaces/community#2551] [Announcements]
![primestore-summersale-2024-o](https://github.com/user-attachments/assets/9bf1aa05-8cf6-4bf6-88fe-af076410e78b)

Dive into summer savings with our amazing Summer Sale at PrimeStore! From now until September 15th, 2024, you can get up to 50% off on premium templates, PrimeBlocks, the Figma UI Kit, and LTS licenses.

Whether you use PrimeFaces, PrimeNG, PrimeReact, or PrimeVue, now’s the perfect time to grab top-tier items at unbeatable prices.

Don’t wait – visit [PrimeStore](https://www.primefaces.org/store/) today and take advantage of these fantastic deals!

**Comment by jmcgillst** [comment]
I purchased PrimeBlocks for PrimeNG yesterday, is there any chance that this discount could be applied retroactively?

**Reply by onursenture** [reply]
Thank you for your purchase. I'm not sure if we can accommodate your request, but please send an email to contact ~at~ primetek.com.tr for further assistance.

**Comment by phuongdk** [comment]
Why I don't see the sale at "https://www.primefaces.org/store/"? Or do I have to put a discount code? I want to purchase some templates at discount prices. Please help!

**Reply by onursenture** [reply]
I checked the store and didn't see any issues with the discount. Could you try using a different browser? If the problem persists, could you send a screenshot?

**Reply by phuongdk** [reply]
Oh is this already applied?
For example, the "freya template" goes from $59 -> $39 -> has the discount already been applied?. I thought 50% sale should be around $30...

**Reply by onursenture** [reply]
Yes, $39 is the discounted price of the Freya template.

**Comment by kehwar** [comment]
The new Primeblocks service has launched and as a previous PrimeBlocks user I got an email to generate a code for a free 1 year sub
How long do I have to generate and use the coupon?

**Comment by michlloenneberga** [comment]
Hey there! I wanted to buy the Figma UI Kit, but I missed it. Is it still possible to get the discount? It would be so lovely! Thank you in advance
